src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx

```diff
@@ -165,91 +170,91 @@ export const CreateNSTreeModal = ({
+              placement="top"
+            >
+              <StyledInfoOutlinedIcon />
+            </StyledTooltip>
+          </TreeNameInfoWrapper>
+          <RadioGroup
+            value={treeType}
+            onChange={(e) => setTreeType(e.target.value as TreeType)}
+          >
+            <StyledFormControlLabel
+              value={TreeTypes.Targeted}
+              checked={treeType === TreeTypes.Targeted}
+              control={<StyledRadio />}
+              label={
+                <RadioLabelTargeted
+                  selected={treeType === TreeTypes.Targeted}
                 />
-              </RadioGroup>
-            </TreeTypeSection>
-            {usesFeatureFlag(FEATURE_FLAGS.gisaidIngest) && (
-              <>
-                <Separator marginSize="l" />
-                <SampleIdInput
-                  handleInputModeChange={handleInputModeChange}
-                  handleInputValidation={handleInputValidation}
-                  shouldReset={shouldReset}
+              }
+            />
+            <StyledFormControlLabel
+              value={TreeTypes.NonContextualized}
+              checked={treeType === TreeTypes.NonContextualized}
+              control={<StyledRadio />}
+              label={
+                <RadioLabelNonContextualized
+                  selected={treeType === TreeTypes.NonContextualized}
                 />
-                <Separator marginSize="xl" />
-              </>
-            )}
-            <MissingSampleAlert missingSamples={missingInputSamples} />
-            <FailedSampleAlert numFailedSamples={failedSamples?.length} />
-            {usesFeatureFlag(FEATURE_FLAGS.gisaidIngest) && (
-              <CreateTreeButton
-                hasValidName={hasValidName}
-                hasSamples={allValidSamplesForTreeCreation.length > 0}
-                isInEditMode={isInputInEditMode}
-                isValidTreeType={Object.values(TreeTypes).includes(treeType)}
-              />
-            )}
-            {!usesFeatureFlag(FEATURE_FLAGS.gisaidIngest) && (
-              <StyledButton
-                color="primary"
-                variant="contained"
-                isRounded
-                disabled={isTreeBuildDisabled}
-                type="submit"
-                value="Submit"
-              >
-                Create Tree
-              </StyledButton>
-            )}
-          </form>
-          <CreateTreeInfo>
-            Creating a new tree can take up to 12 hours.
-          </CreateTreeInfo>
-        </Content>
+              }
+            />
+          </RadioGroup>
+        </TreeTypeSection>
+        {usesFeatureFlag(FEATURE_FLAGS.gisaidIngest) && (
+          <>
+            <Separator marginSize="l" />
+            <SampleIdInput
+              handleInputModeChange={handleInputModeChange}
+              handleInputValidation={handleInputValidation}
+              shouldReset={shouldReset}
+            />
+            <Separator marginSize="xl" />
```

Comment:
is there a way to set marginSize in an emotion styled component?

Reply:
I'm not sure what you mean. Separator is an emotion styled component. I guess I could make two different components with different names, but the only thing that varies is the margin, so it seemed fine to pass it in.